src/Symfony/Component/Messenger/Bridge/Redis/Tests/Transport/ConnectionTest.php

```diff
@@ -441,6 +441,16 @@ public function testDeleteAfterAck()
         $connection->ack('1');
     }
 
+    /**
+     * @group legacy
+     */
+    public function testLegacyOmitDeleteAfterAck()
+    {
+        $this->expectDeprecation('Since symfony/redis-messenger 5.4: Not setting the "delete_after_ack" boolean option explicitly when using the redis transport is deprecated, its default value will change to true in 6.0.');
```

Comment:
```suggestion
        $this->expectDeprecation('Since symfony/redis-messenger 5.4: Not setting the "delete_after_ack" boolean option explicitly when using the Redis transport is deprecated, its default value will change to true in 6.0.');
```